src/main/java/fit/challenge/ImcCalculador.java

```diff
@@ -0,0 +1,24 @@
+package main.java.fit.challenge;
+import java.util.*;
+import java.lang.Math;
+import java.lang.*;
+
+public class ImcCalculador {
+    
+    public static String Calculador(double peso, double altura) {

```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.